db/migrate/20220801144607_create_contact_details_table.rb

```diff
@@ -0,0 +1,12 @@
+class CreateContactDetailsTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :contact_details, id: :uuid do |t|
+      t.timestamps
+
+      t.references :person, type: :uuid, foreign_key: true, null: false, index: { unique: true }
+
+      t.json :home_address, default: {}
```

Comment:
Have you thought about jsonb for this I think we should be able to run faster / more meaningful queries on it if need be?

https://blog.saeloun.com/2022/01/04/postgresql-data-types-in-rails.html#:~:text=Though%20both%20JSON%2FJSONB%20look,querying%20more%20efficient%20and%20faster.

Reply:
I've used `jsonb` in the past for audit tables where queries are performed, but for this I didn't think we would need to query the data (as in doing selects, etc.), but as I don't think there is a negative impact to use `jsonb` even if we don't fully use the capabilities, I can change it 👍